pkg/remote/aws/repository/ecr_repository.go

```diff
@@ -41,3 +42,12 @@ func (r *ecrRepository) ListAllRepositories() ([]*ecr.Repository, error) {
 	r.cache.Put("ecrListAllRepositories", repositories)
 	return repositories, nil
 }
+
+func (r *ecrRepository) GetRepositoryPolicy(repo *ecr.Repository) (*ecr.GetRepositoryPolicyOutput, error) {
+	var repositoryPolicyInput *ecr.GetRepositoryPolicyInput = &ecr.GetRepositoryPolicyInput{
+		RegistryId:     repo.RegistryId,
+		RepositoryName: repo.RepositoryName,
+	}
+
+	return r.client.GetRepositoryPolicy(repositoryPolicyInput)
+}
```

Comment:
This method misses a cache hit, is this intentional ?

Reply:
Also I think we should use a more common return type like `[]*ecr.RepositoryPolicy` if such exists

Reply:
IIRC it does not exist so out of pure laziness I suggested @almog27 we just leave the output type from the request 😅